matplotlibrc.template

```diff
@@ -35,6 +35,9 @@
 # You can also deploy your own backend outside of matplotlib by
 # referring to the module name (which must be in the PYTHONPATH) as
 # 'module://my_backend'.
+#
+# If you omit this parameter, it will always default to "Agg", which is a
```

Comment:
I do not think this is not quite true, the value of `$TEMPLATE_BACKEND` is set at build time based on what it finds.  Some of the wheels we have released do in fact default to Agg (because they are built in very clean environments).